supply/lib/supply/setup.rb

```diff
@@ -1,6 +1,6 @@
 module Supply
   class Setup
-    def perform_download
+    def perform_download(image_format = "png", remove_alpha = false)
```

Comment:
Why these params here? 
Unless I missed something, these are not really used but the default values are applied?

Reply:
I use `Spaceship` directly most of the time (don't have much experience with fastfiles). Having these optional params for users like myself would be extremely useful, as I can simply change them. 

And, being default params, they default way will be applied for users not using them :)